### ISSUE TITLE
remove images/avatars from modules list

### ISSUE THREAD

**Issue by countless-integers** [zendframework/modules.zendframework.com#369]
As much as I appreciate the work that they put into creating module,  I think that this site can do without developer avatars by the modules list. On the modules info page, "new developers" box -- there it's fine, on modules list it's unnecessary. It also makes it harder to view the page on phone as the pics take a lot of space there. I've been showcasing the site to people I'm trying to introduce to ZF2 and it almost invariably weirds them out.

**Comment by ins0** [comment]
i agree on mobile the avatar logos are to big and make it hard to navigate through the page

**Comment by gianarb** [comment]
@countless-integers drop it is drastic! :imp: 
We can work for a best implementation for mobile.. 
@alexz707 proposed a new graphic implementation, maybe you can help us in this improvement :)

**Comment by localheinz** [comment]
@countless-integers 

I agree! How about this:

* Anywhere modules are listed, the avatars are removed - the focus should be on the modules themselves, not the author or organisation which registered them.
* We provide a user profile page, similar to what we have already at http://modules.zendframework.com/user, but which allows to see all the modules a user has registered, e.g., http://modules.zendframework.com/user/countless-integers

If we did that, we would have lighter module lists (no images), but would still have a possibility for users to show-case the modules that they have registered!

**Comment by Ocramius** [comment]
You could (!) in theory parse out the first image from the markdown, but that sounds overkill right now :-)

**Comment by localheinz** [comment]
@Ocramius 

I'm happy to make changes to the layout (as pointed out on IRC and here), but figured it would be cool to sort out the existing bugs beforehand!

**Comment by gianarb** [comment]
In this moment the priority is the production :) For me we can move the layout improvements in another step.. :)

**Comment by Ocramius** [comment]
I wouldn't stop devs that are able to work from doing their job because of
that, especially if they cannot help out directly with the issue.

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/
